mta/README.md

```diff
@@ -0,0 +1,47 @@
+# MTA - migration analysis workflow
+
+# Synopsis
+This workflow is an assessment workflow type, that invokes an application analysis workflow using MTA[1]
+and optionally returns the [move2kube](./m2k) workflow reference, to run next if the analysis is considered to be successful.
+
+Users are encouraged to use this workflow as self-service alternative for interacting with the MTA UI. Instead of running
+a mass-migration of project from a managed place, the project stakeholders can use this (or automation) to regularly check
+the cloud-readiness compatibility of their code.
+
+# Inputs
+- `repositoryUrl` [mandatory] - the git repo url to examine
+- `backstageUserOrGroup` [optional] - the backstage user or group to send backstage notification with the analysis results
+
+# Output
+1. On completion the workflow returns an options[2] structure in the exit state of the flow(also named variables in SonataFlow)
+linking to the [move2kube][3] workflow that will generate k8s manifests for container deployment.
+1. When the workflow completes there should be a report link on the exit state of the flow (also named variables in SonataFlow)
```

Comment:
The report link is currently only available through the notifications. There isn't another way for the user to see it.

Reply:
It was written to the context of the workflow, and was in the 'variables' section in the workflow instance run page. 
I think the section of that writes the output option overwrote that - you can open a bug for it .

@anludke was it necessary to remove "+=" in #37 so the rendering in the ui works?

Reply:
'variables' was renamed to 'results' I see

Reply:
Created bug for this: https://issues.redhat.com/browse/FLPATH-1041